### DISCUSSION TITLE
Release v1.8.3

### DISCUSSION THREAD

**Discussion by pubpub-zz** [pubpub-zz/ppInk#196] [Announcements]
ROB : fix exit pointer when  clicking on other application(#195)
ROB : cursor not correct when exiting pointer #195(part 2 of the issue)

<hr /><em>This discussion was created from the release <a href='https://github.com/pubpub-zz/ppInk/releases/tag/'>Release v1.8.3</a>.</em>

**Comment by ddx23** [comment]
Hi, not sure how else to contact: is there a way to capture menus or tool tips with this app? Menus disappear when I activate the app in any way (e.g. I open an application menu, then use the shortcut to start PPInk --> menu disappears; I have PPInk in pointer mode, try to switch to inking --?> menu disappears). This actually happens sometimes with regular Snip&Sketch (especially tool tips) so one can use timed capture to prevent that. Any way to activate PPInk on a timer (e.g. in 3 seconds, or 5 seconds)

**Comment by pubpub-zz** [comment]
Hi @ddx23 
From what I understand you want to capture menus from your application using ppInk. This feature is called "snapshots in pointer mode" (implementation tracked in #65). You can find a video showing the feature in the https://github.com/pubpub-zz/ppInk, looking for "snapshots in pointer mode"

**Comment by Betty-ai-suguru** [comment]
[ppinkS.zip](https://github.com/pubpub-zz/ppInk/releases/download/v1.8.3/ppinkS.zip)
こちらをダウンロードさせていただきましたが、
MT4にはどのようにして使用するのですか？

**Reply by pubpub-zz** [reply]
in english please

**Reply by pubpub-zz** [reply]
@Betty-ai-suguru any update?

**Comment by 52HZAlice** [comment]
Hello author, during the process of using the software, I found that after making changes in the software settings, the changes made after restarting the software cannot be saved

**Reply by pubpub-zz** [reply]
Did you press save to file in the options?

**Reply by 52HZAlice** [reply]
Oh my god, I'm sorry, but after selecting Chinese on the page, there are still some options that haven't been translated into Chinese. Thank you for your reply

**Reply by pubpub-zz** [reply]
If you are ok, can you complete the translation ?

**Reply by 52HZAlice** [reply]
Hello, as I do not know how to edit software, I have created a PDF file that contains a Chinese translation of general pages. You can refer to it, but I do not understand the meaning of a "magnetic effect" function in it, so it uses machine translation.

**Comment by 52HZAlice** [comment]
戴上我的豌豆射手
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "pubpub-zz/ppInk"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月22日(星期三) 上午6:01
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [pubpub-zz/ppInk] Release v1.8.3 (Discussion #196)





 
If you are ok, can you complete the translation ?

**Comment by steveviuv** [comment]
very nice software, but i have a ting question, how to stick my draw on screen when i press cancel key,thank for your response.

**Reply by pubpub-zz** [reply]
Thanks for comment, do not hesitate to star the project 😀
In the options/hotkeys, the cancel(esc) key assignment can be changed.
Also you can restore your previous drawings using the load button (on the right of the toolbar) if the toolbar has been closed by mistake
